students/PhongT/lesson02/fizzBuzz.py

```diff
@@ -0,0 +1,27 @@
+"""
+Lesson 02
+fizz buzz Exercise
+"""
+
+
+def printFizzBuzz():
```

Comment:
underscores for naming in python, no camel case allowed :) https://www.python.org/dev/peps/pep-0008/